tests/integration/config/plugins/integration-tests/data/tests/tests_illegal_atrocity.txt

```diff
@@ -0,0 +1,240 @@
+				remove "enforces"
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			"%TEST%: SCANNING ILLEGALS"
+
+
+ship "Quarg Wardragon" "%TEST%: Dumbdragon"
+	outfits
+		"Antimatter Core"
+		"Medium Graviton Steering"
+		"Medium Graviton Thruster"
+		"Nanotech Battery"
+	add attributes
+		"outfit scan power" 1000000000
+		"outfit scan speed" 1000000000
+
+mission "%TEST%: Wardragon Scanning You!"
+	priority
+	landing
+	to offer
+		has "%TEST%: SCANNING ILLEGALS"
+	on offer
+		conversation
+			"You must be scanned or be killed."
+				accept
+	npc assist
+		government "Quarg"
+		# I used another definition that had 1 million scan power but whatever
+		ship "%TEST%: Dumbdragon" "Scanner"
+		personality derelict surveillance target
+
+test "Land On Ruin helper"
+	status partial
+	description "Helper for landing on Ruin."
+	sequence
+		input
+			command land
+		label landing
+		branch landing
+			"flagship planet: Ruin" < 1
+		assert
+			"flagship planet: Ruin" > 0
+
+test "Illegal Test - Ignore and New Illegal"
+	status active
+	description "Test if a Quarg ship spawning disabled in the system will give you a fine for an illegal outfit and ignore the fine for another, after you repaired it."
+	sequence
+		inject "quarg scanning"
+		call "Load First Savegame"
+		watchdog 12000
+		assert
+			"fines" == 0
+		call "Depart"
+		# land again to trigger the mission
+		call "Land On Ruin helper"
+		input
+			command "enter"
+		call "Depart"
+		# board the ship so we're sure we get close to it to get scanned
+		input
+			command board
+		# Wait for 10 seconds so it should be repaired
+		apply
+			"test: steps to wait" = 10 * 60
+		label "floating"
+		# empty input to make time pass
+		input
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch "floating"
+			"test: steps to wait" > 0
+		# skip the dialog message for the fine
+		input
+			command "enter"
+		# land back with a fine of 1
+		call "Land On Ruin helper"
+		assert
+			"fines" == 1
+
+test "Atrocity Test - New Atrocity"
+	status active
+	description "Test if a Quarg ship spawning disabled in the system will consider an outfit an atrocity when specified by an event."
+	sequence
+		inject "quarg atrocity"
+		call "Load First Savegame"
+		watchdog 12000
+		assert
+			"reputation: Quarg" == 1000
+		call "Depart"
+		# land again to trigger the mission
+		call "Land On Ruin helper"
+		input
+			command "enter"
+		call "Depart"
+		# board the ship so we're sure we get close to it to get scanned
+		input
+			command board
+		# Wait for 10 seconds so it should be boarded
+		apply
+			"test: steps to wait" = 10 * 60
+		label "floating"
+		# empty input to make time pass
+		input
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch "floating"
+			"test: steps to wait" > 0
+		# skip the dialog message for the fine
+		input
+			command "enter"
+		# land back with a fine of 1
+		call "Land On Ruin helper"
+		assert
+			"reputation: Quarg" < 1
```

Comment:
Checking the reputation is probably a good way to detect that the atrocity was registered. (But I don't see why this test is also taking steps to get the fine.)

Reply:
Its the other test that is about the fine this one is for the atrocity